apps/api/src/expenses/expenses.controller.ts

```diff
@@ -11,12 +11,19 @@ export class ExpensesController {
   constructor(private readonly expensesService: ExpensesService) {}
 
   @Public()
```

Comment:
It should be either `@Public()` or `@Roles()`

I'm not certain about the behavior of both together.

```suggestion
```